src/structures/Invite.js

```diff
@@ -71,22 +72,31 @@ class Invite extends Base {
     this.inviter = data.inviter ? this.client.users.add(data.inviter) : null;
 
     /**
-     * The target user for this invite
+     * The target user of this invite
```

Comment:
```suggestion
     * The user whose stream to display for this voice channel stream invite
```
from https://discord.com/developers/docs/resources/invite#invite-object-invite-structure